control/freqplot.py

```diff
@@ -173,19 +170,26 @@ def bode_plot(syslist, omega=None,
     Examples
     --------
     >>> sys = ss("1. -2; 3. -4", "5.; 7", "6. 8", "9.")
-    >>> mag, phase, omega = bode(sys)
+    >>> bode_plot(sys)
 
     """
+
     # Make a copy of the kwargs dictionary since we will modify it
     kwargs = dict(kwargs)
 
-    # Check to see if legacy 'Plot' keyword was used
+    # Check to see if legacy 'Plot' and 'plot' keywords were used
     if 'Plot' in kwargs:
         import warnings
-        warnings.warn("'Plot' keyword is deprecated in bode_plot; use 'plot'",
-                      FutureWarning)
-        # Map 'Plot' keyword to 'plot' keyword
-        plot = kwargs.pop('Plot')
+        warnings.warn("'Plot' keyword is deprecated in bode_plot. Use " \
+                      "frequency_response_bode to generate mag, phase, " \
+                      "omega data.", FutureWarning)
```

Comment:
The backslashes are not required inside the parantheses of the `warn()` function.

While at it, the warning should include an appropriate stacklevel parameter, so that you see the calling function using the deprecated parameter. And it should actually be a `DeprecatedWarning`.